docs/src/development.md

```diff
@@ -1,16 +1,62 @@
 # Development
-The `License Manager` application incorporates a mix of different services in docker and LXD containers.
-This text will attempt to define the procedure for initializing and running the different components.
+The `License Manager` can be provisioned using two approaches:
 
-## Pre-Installation
+* Use `Docker Compose` to run the services
+* Create a `Slurm` cluster using Juju
+
+## Using Docker Compose
+
+### Pre-Installation
+Before you get started, ensure you have the following pre-requisites installed on your machine:
+
+* docker
+* docker compose
+
+### Running the License Manager Composed
+To get started, clone the `license-manager` repository from GitHub and run `docker compose up`.
+
+``` bash
+git clone https://github.com/omnivector-solutions/license-manager
+cd license-manager/lm-composed
+docker-compose up --build
+```
+
+The `docker compose` command will start the following services:
+
+1. License Manager API
+2. Keycloak (authentication provider for the-LM API)
+3. License Manager Simulator API
+4. Slurm cluster (with two compute node)
+
+### Submitting a job
+1. Log into the `slurmctld` container:
+
+```bash 
+docker exec -it slurmctld bash
```

Comment:
Since we are using docker compose, we should be using using docker compose commands here:
```bash
docker compose exec slurmctld bash
```

Reply:
Thanks, I'd been using the other command but this one is better.